settings.gradle.kts

```diff
@@ -6,7 +6,7 @@ pluginManagement {
     }
 }
 dependencyResolutionManagement {
-    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+//    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
```

Comment:
This can be reverted, we still want that line

Reply:
`js` target imports libraries dynamically from the plugin it self  (npm packages)

So build was failing with this line, 
I tried using preferred settings , also failed so i commented

Reply:
True, I also get an error now. Then we can delete this line.